src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars */
+
+// ACTIONS
+export function addBook() {
+  return {
+    type: 'ADD_BOOK',
+  };
+}
+
+export function removeBook() {
+  return {
+    type: 'REMOVE_BOOK',
+  };
+}
+
+// REDUCER
+export default function reducer(state, action) {
```

Comment:
- [ ] Please, use ES6 best practices, in this case, this could improve being an arrow function. 🦊